packages/trimerge-sync/src/CoordinatingLocalStore.ts

```diff
@@ -458,12 +462,18 @@ export class CoordinatingLocalStore<CommitMetadata, Delta, Presence>
   }
 
   private initialize(): Promise<void> {
-    const { getRemote, networkSettings, initialized } = this;
+    const { remote, networkSettings, initialized } = this;
     if (initialized) {
       throw new Error('only call initialize() once');
     }
     this.initialized = true;
-    if (getRemote) {
+    if (remote) {
+      // If we have a remote, we always listen but if we're not the leader, we won't connect.
+      remote.listen((event) => {
```

Comment:
Are non-leaders going to get all the messages they expect with the new event buffer?  Do non-leaders break the invariant of one listener allowed?